doc/cprover-manual/memory-analyzer.md

```diff
@@ -0,0 +1,70 @@
+[CPROVER Manual TOC](../../)
+
+## Memory Analyser
+
+The memory analyser is a front for running and querying GDB about a state of
```

Comment:
is a front for -> is a front-end used for

Reply:
Changed.